### PR TITLE
Update OTel dependency to 0.15.0

### PR DESCRIPTION
**Pull request by arminru**
Includes some improvements to MetricAdapter

### CODE REVIEW

**Comment by Oberon00** [comment]
[0.16.0](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v0.16.0) was just released and is probably closer to 1.0.

File: dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/MetricAdapter.java [hunk 1]
```diff
@@ -36,13 +42,15 @@
...
+  private MetricAdapter() {}
+
   /**
    * Generates a MintLineProtocolSerializable MintMetricsMessage, that can be ingested to MINT.
    *
    * @param metrics are the OT metrics generated by metric instruments.
    * @return the MintMetricsMessage, that can be serialized to MINT compatible String.
    */
-  static MintMetricsMessage toMint(Collection<MetricData> metrics) {
+  public static MintMetricsMessage toMint(Collection<MetricData> metrics) {
```
Comment (thisthat) [inline dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/MetricAdapter.java:53]: I don't think we should export this method to the world.

File: dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/MetricAdapter.java [hunk 2]
```diff
@@ -62,98 +70,150 @@ static MintMetricsMessage toMint(Collection<MetricData> metrics) {
   static Collection<Datapoint> toDatapoints(MetricData metric) {
     Collection<Datapoint> datapoints = new ArrayList<>();
 
-    Collection<MetricData.Point> points = metric.getPoints();
-
-    if (points.isEmpty()) { // necessary because sometimes an empty MetricData is ingested by OT
+    if (metric.isEmpty()) { // necessary due to sometimes an empty MetricData is ingested by OT
       return datapoints;
```
Comment (thisthat) [inline dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/MetricAdapter.java:74]:
```suggestion
      return Collections.emptyList();
```

File: dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/NormalizationTest.java [hunk 3]
```diff
@@ -19,17 +19,20 @@
...
 import org.junit.jupiter.api.Test;
 
 public class NormalizationTest {
-  private static final Meter meter =
-      OpenTelemetry.getGlobalMeterProvider().get("test$lib", "test:1.0.0");
+  private static final SdkMeterProvider provider =
+      SdkMeterProvider.builder().buildAndRegisterGlobal();
+  private static final Meter meter = provider.get("test$lib", "test:1.0.0");
+
   //  private static final DoubleSumObserver doubleSumObserver =
```
Comment (thisthat) [inline dynatrace/src/test/java/com/dynatrace/opentelemetry/metric/NormalizationTest.java:36]: I think these comments can be removed

**Review by thisthat — APPROVED** [review]

**Pull request merged**